docs/javascript/exercises/5.md

```diff
@@ -86,7 +86,13 @@ The only thing we are doing in this file is including the `exercise5` module we
 Bundles can do more as well.
 Then can include initialization code for example.
 
+```{todo}
+The following statement is no longer true.
+The link is broken and needs to be updated.
+See https://github.com/plone/plone.staticresources/issues/213
+
 See Plone's [default bundle](https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/plone.js).
```

Comment:
```suggestion
See Plone's [default bundle](https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/static/).
```
This fixes the broken link.

Reply:
Oh, sorry, I forgot to comment. Is that the correct link? I ask because the text says "Plone's default bundle", but there are three bundles at that link, [bundle-bootstrap](https://github.com/plone/plone.staticresources/tree/master/src/plone/staticresources/static/bundle-bootstrap), [bundle-jquery](https://github.com/plone/plone.staticresources/tree/master/src/plone/staticresources/static/bundle-jquery), and [bundle-plone](https://github.com/plone/plone.staticresources/tree/master/src/plone/staticresources/static/bundle-plone) itself.

And there is an open issue related to this broken link: https://github.com/plone/plone.staticresources/issues/213

Reply:
Hi @petschki Can you please enlighten us: We try to fix broken links in the "Javascript" training. I suppose it would be good to keep the training as far as possible for Plone 5. How can we maintain it concerning the links targeting resources in packages like staticresources, mockup, etc?

Reply:
For Plone 5 you have to stay on `1.x` branch in `plone.staticresources` ... see https://github.com/plone/plone.staticresources/blob/1.x/src/plone/staticresources/static/plone.js

the issue for upgrading `README.txt` is meant as documentation for ES6 module implementation for addons etc ... we'll work at the Buschenschank sprint on that.

Reply:
Thank you @petschki! I replaced the example with one that has some initialization. Hope this fits.

Reply:
Done!